File: AIS/AIS.BLL/Models/Session.cs
using System;
using System.Collections.Generic;
using AIS.BLL.Interfaces;

namespace AIS.BLL.Models
{
    public class Session : IHasIdBase
    {
        public int Id { get; set; }
        public DateTime StartedAt { get; set; }
        public int CompanyId { get; set; }
        public Company Company { get; set; }
        public Employee Employee { get; set; }
        public int EmployeeId { get; set; }
        public Interviewee Interviewee { get; set; }
        public int IntervieweeId { get; set; }
        public DateTime FinishedAt { get; set; }
        public int QuestionAreaId { get; set; }
        public QuestionArea QuestionArea { get; set; }
        public ICollection <QuestionIntervieweeAnswer> questionIntervieweeAnswers { get; set; }
    }
}

File: AIS.front/ais.front/src/core/interfaces/session.ts
import { Company } from "./company";
import { Employee } from "./employee";
import { Interviewee } from "./interviewee";
import { QuestionArea } from "./questionArea";
import { QuestionIntervieweeAnswer } from "./questionIntervieweeAnswer";

export interface Session{
    id: number,
    startedAt: string,
    companyId: number,
    company: Company | null,
    employeeId: number,
    employee: Employee | null,
    intervieweeId: number,
    interviewee:Interviewee | null,
    finishedAt: string,
    questionAreaId: string,
    questionArea: QuestionArea | null,
    questionIntervieweeAnswers: QuestionIntervieweeAnswer | null
}
